aeutils_eqc/aeu_mp_trees_eqc.erl

```diff
@@ -44,7 +44,10 @@ gen_subtree_key(Existing) ->
 
 gen_value() ->
     frequency([{1, <<>>},
-               {9, weighted_default({5, gen_decodable()}, {1, binary(36)})}]).
+               {9, gen_non_empty_value()}]).
```

Comment:
Please note that this portion of the definition of the API of the Erlang module has been there since [the initial version](https://github.com/aeternity/aeternity/pull/605/files#diff-e18e2f92797a228401093ff993dfcad9R84).

Reply:
Please also see:
- Resetting semantics of AEVM op SSTORE.
- [AEVM SSTORE op implementation](https://github.com/aeternity/aeternity/blob/9e251f87532934df535f315c265ac9887fd6fc6a/apps/aevm/src/aevm_eeevm_store.erl#L427).
- [Handling of contract store state tree](https://github.com/aeternity/aeternity/blob/9e251f87532934df535f315c265ac9887fd6fc6a/apps/aecontract/src/aect_state_tree.erl#L125).